st2chatops.env

```diff
@@ -42,7 +42,7 @@ export ST2_WEBUI_URL="${ST2_WEBUI_URL:-https://${ST2_HOSTNAME}}"
 # Chat service adapter settings
 
 # Uncomment one of the adapter blocks below.
-# Currently supported: slack, hipchat, xmpp, matteruser, rocketchat, spark, irc, flowdock.
+# Currently supported: slack, botframework, hipchat, xmpp, flowdock, spark, irc, matteruser, rocketchat.
```

Comment:
```suggestion
# Currently supported: slack, botframework (Microsoft Teams), hipchat, xmpp, flowdock, spark, irc, matteruser, rocketchat.
```
may look better as `botframework` automatically meaning `MS Teams` is not entirely clear.

Reply:
:+1: